mmv1/products/alloydb/Instance.yaml

```diff
@@ -262,38 +262,54 @@ properties:
     description: 'Configuration for enhanced query insights.'
     min_version: 'beta'
     default_from_api: true
+    custom_expand: 'templates/terraform/custom_expand/alloydb_instance_observability_config.go.tmpl'
     properties:
       - name: 'enabled'
         type: Boolean
         description: 'Observability feature status for an instance.'
         include_empty_value_in_cai: true  # Default value is false in CAI asset
+        send_empty_value: true
+        default_from_api: true
       - name: 'preserveComments'
         type: Boolean
         description: 'Preserve comments in the query string.'
         include_empty_value_in_cai: true  # Default value is false in CAI asset
+        send_empty_value: true
+        default_from_api: true
       - name: 'trackWaitEvents'
         type: Boolean
         description: 'Record wait events during query execution for an instance.'
+        send_empty_value: true
+        default_from_api: true
       - name: 'trackWaitEventTypes'
         type: Boolean
         description: 'Record wait event types during query execution for an instance.'
+        output: true
```

Comment:
This is a breaking change and can't be made in a minor version. What happened here that this isn't settable?